site/src/components/UserAvatar/UserAvatar.stories.tsx

```diff
@@ -0,0 +1,24 @@
+import type { Meta, StoryObj } from "@storybook/react";
+import { UserAvatar } from "./UserAvatar";
+
+const meta: Meta<typeof UserAvatar> = {
+  title: "components/UserAvatar",
+  component: UserAvatar,
+};
+
+export default meta;
+type Story = StoryObj<typeof UserAvatar>;
+
+export const Jon: Story = {
+  args: {
+    username: "sreya",
+    avatarURL: "https://github.com/sreya.png",
+  },
+};
+
+export const JonButJapanese: Story = {
+  args: {
+    username: "ジョン",
```

Comment:
I like where your head is at!

Reply:
Also just realized: it's proper to use the kagikakko quotation marks because this would be the title of a book, but most people are also going to recognize them as "Jojo quotes"